### PR TITLE
Added release notes for 5.10

### PR DESCRIPTION
**Pull request by dennisdoomen**

### CODE REVIEW

File: docs/_pages/releases.md [hunk 1]
```diff
@@ -7,6 +7,39 @@ sidebar:
...
+* Several assertion APIs did not include the _reason_ in the failure messages - [#1172](https://github.com/fluentassertions/fluentassertions/pull/1172)
+* A self-closing XML element was not treated as equivalent to an empty element with the same name - [#1174](https://github.com/fluentassertions/fluentassertions/pull/1174)
+* Ensure that type assertion `ThatAreUnderNamespace` handles types in the global namespaces correctly - [#1197](https://github.com/fluentassertions/fluentassertions/pull/1197)
+* When comparing whether two objects are equal, a conversion should be precision preserving - [#1202](https://github.com/fluentassertions/fluentassertions/pull/1202)
+* `BeEquivalentTo` on normal tuples should use structural equivalency instead of treating it as a value type - [#1206](https://github.com/fluentassertions/fluentassertions/pull/1206)
+* Some platforms throw reflection exceptions when trying to use `ConfigurationManager`. This is now handled more gracefully - [#1210](https://github.com/fluentassertions/fluentassertions/pull/1210)
+* Reintroduced the package dependency on `System.Xml.Linq` for .NET 4.5/4.7
```
Comment (jnyrup) [inline docs/_pages/releases.md]: Do you want a link to a PR or an issue?

File: docs/_pages/releases.md [hunk 2]
```diff
@@ -7,6 +7,39 @@ sidebar:
...
+* Fixed a crash that can occur when trying to determine the caller identity under .NET Native - [#1149](https://github.com/fluentassertions/fluentassertions/issues/1149)
+* Ensured that determining the caller identity works for namespaces that contain the phrase `System` - [#1193](https://github.com/fluentassertions/fluentassertions/pull/1193)
+* `BeEquivalentTo` on an `XDocument` could report the incorrect path to a self-closing XML tag - [#1170](https://github.com/fluentassertions/fluentassertions/pull/1170)
+* `BeEquivalentTo` on multi-dimensional arrays with empty elements could cause an internal error - [#1167](https://github.com/fluentassertions/fluentassertions/issues/1167)
+* Several assertion APIs did not include the _reason_ in the failure messages - [#1172](https://github.com/fluentassertions/fluentassertions/pull/1172)
+* A self-closing XML element was not treated as equivalent to an empty element with the same name - [#1174](https://github.com/fluentassertions/fluentassertions/pull/1174)
+* Ensure that type assertion `ThatAreUnderNamespace` handles types in the global namespaces correctly - [#1197](https://github.com/fluentassertions/fluentassertions/pull/1197)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: The main fix in that PR is fixing comparing partial namespaces.
E.g. `Internal.Main.Tes` no longer considered being under namespace `Internal.Main.Test`.

File: docs/_pages/releases.md [hunk 3]
```diff
@@ -7,6 +7,39 @@ sidebar:
...
 
+## 5.10.0
+
+**What's New**
+* Added `string.Should().NotBeEquivalent` - [#1134](https://github.com/fluentassertions/fluentassertions/pull/1134)
+* Added `collectionOfStrings.Should().ContainMatch()` to assert that the collection contains at least one string that matches a wildcard pattern - [#1138](https://github.com/fluentassertions/fluentassertions/pull/1138)
+* Added overloads of `NotBeInAscendingOrder` and `NotBeInDescendingOrder` that take a property expression - [#1140](https://github.com/fluentassertions/fluentassertions/pull/1140)
+* Include the index in the XPath information reported while comparing `XDocument`s for equivalence - [#1181](https://github.com/fluentassertions/fluentassertions/pull/1181)
+* Added `NotHaveSameCount` and `HaveSameCount` for dictionary assertions - [#1178](https://github.com/fluentassertions/fluentassertions/pull/1178)
+* Extended `string.Should().Contain()` and `string.Should().ContainEquivalentOf()` to test the number of times a phrase exists in a string using `theString.Should().Contain("is a", MoreThan.Thrice());` - [#1145](https://github.com/fluentassertions/fluentassertions/pull/1145)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: Remove `;`?

File: docs/_pages/releases.md [hunk 4]
```diff
@@ -7,6 +7,39 @@ sidebar:
   nav: "sidebar"
 ---
 
+## 5.10.0
+
+**What's New**
+* Added `string.Should().NotBeEquivalent` - [#1134](https://github.com/fluentassertions/fluentassertions/pull/1134)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: `NotBeEquivalentTo`

File: docs/_pages/releases.md [hunk 5]
```diff
@@ -7,6 +7,39 @@ sidebar:
...
+* `BeEquivalentTo` on an `XDocument` could report the incorrect path to a self-closing XML tag - [#1170](https://github.com/fluentassertions/fluentassertions/pull/1170)
+* `BeEquivalentTo` on multi-dimensional arrays with empty elements could cause an internal error - [#1167](https://github.com/fluentassertions/fluentassertions/issues/1167)
+* Several assertion APIs did not include the _reason_ in the failure messages - [#1172](https://github.com/fluentassertions/fluentassertions/pull/1172)
+* A self-closing XML element was not treated as equivalent to an empty element with the same name - [#1174](https://github.com/fluentassertions/fluentassertions/pull/1174)
+* Ensure that type assertion `ThatAreUnderNamespace` handles types in the global namespaces correctly - [#1197](https://github.com/fluentassertions/fluentassertions/pull/1197)
+* When comparing whether two objects are equal, a conversion should be precision preserving - [#1202](https://github.com/fluentassertions/fluentassertions/pull/1202)
+* `BeEquivalentTo` on normal tuples should use structural equivalency instead of treating it as a value type - [#1206](https://github.com/fluentassertions/fluentassertions/pull/1206)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: `Tuple`s.
Mix of plural (normal types) and singular (it as a value type)

**Comment by dennisdoomen** [comment]
> And then notes about the (hopefully) soon to be merged PRs.

I'm going to release 5.10 without waiting for those.

File: docs/_pages/releases.md [hunk 6]
```diff
@@ -7,6 +7,40 @@ sidebar:
...
+* Added overloads of `NotBeInAscendingOrder` and `NotBeInDescendingOrder` that take a property expression - [#1140](https://github.com/fluentassertions/fluentassertions/pull/1140)
+* Include the index in the XPath information reported while comparing `XDocument`s for equivalence - [#1181](https://github.com/fluentassertions/fluentassertions/pull/1181)
+* Added `NotHaveSameCount` and `HaveSameCount` for dictionary assertions - [#1178](https://github.com/fluentassertions/fluentassertions/pull/1178)
+* Extended `string.Should().Contain()` and `string.Should().ContainEquivalentOf()` to test the number of times a phrase exists in a string using `theString.Should().Contain("is a", MoreThan.Thrice())` - [#1145](https://github.com/fluentassertions/fluentassertions/pull/1145)
+* Added support for methods returning `ValueTask` and `ValueTask<T>` - [#1158](https://github.com/fluentassertions/fluentassertions/pull/1158)
+* Added support using `SatisfyRespectively` on all collections, including collections of `string` - [#1201](https://github.com/fluentassertions/fluentassertions/pull/1201)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: Added support *for* using?

**Pull request merged**